lib/video_info/providers/vimeo.rb

```diff
@@ -17,8 +21,28 @@ def provider
         define_method(method) { _video[method].to_i }
       end
 
+      def title
+        _video['name']
+      end
+
+      def thumbnail_small
+        "https://i.vimeocdn.com/video/#{video_id}_100x75.jpg"
+      end
+
+      def thumbnail_medium
+        "https://i.vimeocdn.com/video/#{video_id}_200x150.jpg"
+      end
+
+      def thumbnail_large
+        "https://i.vimeocdn.com/video/#{video_id}_640.jpg"
+      end
+
       def keywords
-        _video['tags']
+        keywords_array.join(', ')
+      end
+
+      def keywords_array
+        _video['tags'].map { |t| t["tag"] }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.